ui/front/form.php

```diff
@@ -200,18 +200,20 @@
 	if ( 'undefined' === typeof pods_form_init ) {
 		var pods_form_init = true;
 
-		jQuery(document).ready( function( $ ) {
-			if ( 'undefined' !== typeof jQuery( document ).Pods ) {
+		document.addEventListener( "DOMContentLoaded", function() {
+			jQuery(document).ready( function( $ ) {
```

Comment:
Listening for `DOMContentLoaded` should remove the need for the jQuery ready() block entirely.

Reply:
Sure? I thought jQuery.ready was an event that was triggered at a later stage.

Reply:
The only real difference between listening for that event and using jQuery's ready is that ready will still be called if the code is executed after `DOMContentLoaded` is fired whereas you'd obviously not catch an one-time event that already fired.  Since the code will be enqueued in the body of the document there should be zero risk of that happening and the ready call is just redundant.